public/app/features/alerting/unified/components/rules/central-state-history/EventListSceneObject.tsx

```diff
@@ -39,74 +25,61 @@ import { CollapseToggle } from '../../CollapseToggle';
 import { LogRecord } from '../state-history/common';
 import { useRuleHistoryRecords } from '../state-history/useRuleHistoryRecords';
 
-const LIMIT_EVENTS = 250;
-
-const HistoryEventsList = ({ timeRange }: { timeRange?: TimeRange }) => {
-  const styles = useStyles2(getStyles);
-
-  // Filter state
-  const [eventsFilter, setEventsFilter] = useState('');
-  // form for filter fields
-  const { register, handleSubmit, reset } = useForm({ defaultValues: { query: '' } }); //  form for search field
+import { LABELS_FILTER } from './CentralAlertHistoryScene';
+
+export const LIMIT_EVENTS = 9000;
+
+/**
+ *
+ * This component displays a list of history events.
+ * It fetches the events from the history api and displays them in a list.
+ * The list is filtered by the labels in the filter variable and by the time range variable in the scene graph.
+ */
+export const HistoryEventsList = ({
+  timeRange,
+  valueInfilterTextBox,
+}: {
+  timeRange?: TimeRange;
+  valueInfilterTextBox: VariableValue;
+}) => {
   const from = timeRange?.from.unix();
   const to = timeRange?.to.unix();
-  const onFilterCleared = useCallback(() => {
-    setEventsFilter('');
-    reset();
-  }, [setEventsFilter, reset]);
 
   const {
     data: stateHistory,
     isLoading,
     isError,
     error,
-  } = stateHistoryApi.endpoints.getRuleHistory.useQuery(
-    {
-      from: from,
-      to: to,
-      limit: LIMIT_EVENTS,
-    },
-    {
-      refetchOnFocus: true,
-      refetchOnReconnect: true,
-    }
-  );
+  } = stateHistoryApi.endpoints.getRuleHistory.useQuery({
+    from: from,
+    to: to,
+    limit: LIMIT_EVENTS,
+  });
 
-  const { historyRecords } = useRuleHistoryRecords(stateHistory, eventsFilter);
+  const { historyRecords } = useRuleHistoryRecords(stateHistory, valueInfilterTextBox.toString());
 
   if (isError) {
     return <HistoryErrorMessage error={error} />;
   }
 
   return (
-    <Stack direction="column" gap={1}>
-      <div className={styles.labelsFilter}>
-        <form onSubmit={handleSubmit((data) => setEventsFilter(data.query))}>
-          <SearchFieldInput
```

Comment:
we don't use this filter anymore. We moved the logic of filtering to the scenes variable on the parent scene, as it brings the benefit of sharing the filter between the two children scenes and also the synchronisation with the query params.